.env.example

```diff
@@ -45,9 +45,10 @@ CONVECTION_GEMINI_TEMPLATE=convection-staging
 DEBUG=info,error
 DIFFUSION_REQUEST_THROTTLE_MS=60000
 ARTICLE_REQUEST_THROTTLE_MS=60000
+ENABLE_ASYNC_STACK_TRACES=false
 ENABLE_QUERY_TRACING=false
 ENABLE_REQUEST_LOGGING=true
-ENABLE_ASYNC_STACK_TRACES=false
+ENABLE_SCHEMA_STITCHING=true
```

Comment:
I'm reasonably sure that I removed this

Reply:
Let me see what happens when i remove this -- i recall MP not booting (but I haven't been in here in like a year, could have been something else. I recloned repo.)

Reply:
Cool, all good 👍